### PR TITLE
fix(edgeless): note index doesn't move with viewport

### LINKED ISSUE #3945: Note index doesn't move with viewport
https://github.com/toeverything/blocksuite/assets/7312949/1e9b4818-b908-4454-aeeb-07d8589e85de

### PR DESCRIPTION
**Pull request by tomoyukiy**
fixed #3945 

This is my first pull request in this repository.
So, I would appreciate any feedback I can receive.

https://github.com/toeverything/blocksuite/assets/38880175/108a5936-2992-43f8-aba2-214aef634546

### CODE REVIEW

**Review by doodlewind — COMMENTED** [review]
Could you update [CLA](https://github.com/toeverything/blocksuite/edit/master/.github/CLA.md) for the following review process? Thanks!

**Comment by tomoyukiy** [comment]
@doodlewind 
Created a pull request(#3986) to update the CLA

File: packages/blocks/src/page-block/edgeless/components/notes-status.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import { computePosition, flip, offset } from '@floating-ui/dom';
+import { matchFlavours } from '@blocksuite/store';
 import { html, nothing } from 'lit';
 import { styleMap } from 'lit/directives/style-map.js';
 
-import { getBlockElementById } from '../../../__internal__/utils/query.js';
 import type { NoteBlockModel } from '../../../note-block/note-model.js';
 import type { EdgelessPageBlockComponent } from '../edgeless-page-block.js';
 import { isTopLevelBlock } from '../utils/query.js';
+import { updateNotesPoision } from './utils.js';
```
Comment (fourdim) [inline packages/blocks/src/page-block/edgeless/components/notes-status.ts:9]: Please fix the typo.

**Comment by tomoyukiy** [comment]
@fourdim
Thank you for your review.
Sorry, contains typos.
 I fixed it, so please check it again.

**Review by fourdim — APPROVED** [review]

**Pull request merged**